src/DashboardService.Data/BoardRepository.cs

```diff
@@ -93,7 +93,7 @@ await boards
   // TODO
   // Remove entity or isActive = false?
 
-  public async Task<bool> RemoveAsync(Guid id, CancellationToken ct)
+  public async Task<bool> RemoveAsync(Guid id, CancellationToken ct, IGroupRepository groupRepository)
```

Comment:
May be you can remove in this method, not need in groupRepository?

Reply:
++